src/container/Container.js

```diff
@@ -11,4 +11,4 @@ function Container() {
     )
```

Comment:
Here you can see inconsistence of names on component and it's styles. Your component name is Container but it's class name is containerForm. Please look to next comment below to see description.

Reply:
Changed the div className also to 'formContainer'